docs/core/extensions/generic-host.md

```diff
@@ -63,7 +63,7 @@ The <xref:Microsoft.Extensions.Hosting.Host.CreateApplicationBuilder%2A> method:
   - EventLog (only when running on Windows)
 - Enables scope validation and [dependency validation](xref:Microsoft.Extensions.DependencyInjection.ServiceProviderOptions.ValidateOnBuild) when the environment is `Development`.
 
-The `ConfigureServices` method exposes the ability to add services to the <xref:Microsoft.Extensions.DependencyInjection.IServiceCollection?displayProperty=nameWithType> instance. Later, these services can be made available from dependency injection.
+The <xref:Microsoft.Extensions.Hosting.HostApplicationBuilder.Services?displayProperty=nameWithType> is an <xref:Microsoft.Extensions.DependencyInjection.IServiceCollection?displayProperty=nameWithType> instance. These services are used to build an <xref:System.IServiceProvider> that's used with dependency injection to resolve the registered services.
```

Comment:
```suggestion
<xref:Microsoft.Extensions.Hosting.HostApplicationBuilder.Services?displayProperty=nameWithType> is a <xref:Microsoft.Extensions.DependencyInjection.IServiceCollection?displayProperty=nameWithType> instance. These services are used to build an <xref:System.IServiceProvider> that's used with dependency injection to resolve the registered services.
```

Reply:
I think we should leave this as `an IServiceCollection instance`.

Reply:
Then we need to remove nameWithType